modules/client/src/allowlistVoting/internal/methods.ts

```diff
@@ -37,11 +37,13 @@ export class AllowlistVotingMethods {
   /**
    * Returns the PCT_BASE
    *
-   * @return {*}  {Promise<BigNumber>}
+   * @return {*}  {Promise<number>}
    * @memberof AllowlistVotingMethods
    */
-  public PCT_BASE(): Promise<BigNumber> {
-    return this.allowlistVoting.getPluginInstanceWithSigner().PCT_BASE();
+  public async PCT_BASE(): Promise<number> {
+    return (
+      await this.allowlistVoting.getPluginInstanceWithSigner().PCT_BASE()
+    ).toNumber();
```

Comment:
Can we use a slightly clearer name? :) 
We are calling the concept "precision" and dealing with "digits"

Reply:
done